src/components/error-summary/error-summary.js

```diff
@@ -12,6 +14,103 @@ ErrorSummary.prototype.init = function () {
   window.addEventListener('load', function () {
     $module.focus()
   })
+
+  $module.addEventListener('click', this.onClick.bind(this))
+}
+
+/**
+* Click event handler
+*
+* @param {MouseEvent} event - Click event
+*/
+ErrorSummary.prototype.onClick = function (event) {
```

Comment:
Non-blocking: I think we normally try to use 'handleClick', like the React convention, isn't written down so is OK if you ignore this.